LICENSE.txt

```diff
@@ -0,0 +1,47 @@
+PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
+--------------------------------------------
+
+1. This LICENSE AGREEMENT is between the Python Software Foundation
+("PSF"), and the Individual or Organization ("Licensee") accessing and
+otherwise using this software ("Jython") in source or binary form and
```

Comment:
Since this license refers to the repo at hand, `Jython` should be replaced by `jython-cli` I guess. To make it clear it is about the cli specifically. Jython itself is of no concern here as it has its own license copy in its own repo.

Reply:
(search/replace alls occurrences in this document of course)

Reply:
Renamed Jython to jython-cli